CMSgov/onemac
# OY2-10952: fixing the CSV headers (again)

Story: https://qmacbis.atlassian.net/browse/OY2-10952
Endpoint: https://d274nro68v7qzt.cloudfront.net/

### Details

Fix the CSV export headers.

### Changes

- Re-add "State" header to Dashboard export
- Add trailing commas to the final header names in each export type so last letter doesn't get dropped

### Implementation Notes

N/A

### Test Plan

1. Login as `helpdeskactive` and download the Dashboard as CSV. Verify that the headers are correct and do not have the issues observed in the ticket.
2. Go to the User Management dashboard and repeat step 1.
